tutorials/frontend/from_tflite.py

```diff
@@ -0,0 +1,190 @@
+"""
+Compile TFLite Models
+===================
+**Author**: `Zhao Wu <https://github.com/FrozenGene>`_
+
+This article is an introductory tutorial to deploy TFLite models with Relay.
+
+For us to begin with, Flatbuffers and TFLite package must be installed.
+
+A quick solution is to install Flatbuffers
```

Comment:
I guess you mean "A quick solution is to install Flatbuffers via pip" ? suggest to say "For Flatbuffers it is easy to install via pip"

Reply:
yes. omit some words.